docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -120,7 +120,7 @@ There are optional properties useful to refine your `EntityManagerFactory` or gu
 ==== Dialect
 
 `quarkus.hibernate-orm.dialect`:: (e.g. `org.hibernate.dialect.PostgreSQL95Dialect`).
-Class name of the Hibernate ORM dialect.
+Class name of the Hibernate ORM dialect. Complete list of available dialects values can be seen https://docs.jboss.org/hibernate/orm/5.4/javadocs/org/hibernate/dialect/package-summary.html[here] in the `Class Summary` section. A concrete `dialect` value may then be specified by prefixing a Class name with `"org.hibernate.dialect."` .
```

Comment:
I would go with this:
`The complete list of bundled dialects is available in the https://docs.jboss.org/hibernate/stable/orm/javadocs/org/hibernate/dialect/package-summary.html[Hibernate ORM JavaDoc].`

I would also add a note just at the end of this Dialect paragraph saying:
```
[NOTE]
====
Not all the dialects are supported in GraalVM native executables: we currently provide driver extensions for PostgreSQL, MariaDB and Microsoft SQL Server and H2.
====
```

@machi1990 can you take care of this? Thanks.

Reply:
Done. 

I dropped the extra `"and"` from the note phrase`..PostgreSQL, MariaDB and Microsoft SQL Server and H2.`

Reply:
Good call, I put H2 at the beginning then at the end and forgot to remove the `and`.

Thanks for taking care of it, merging.